src/python/pants/option/options_test.py

```diff
@@ -1530,17 +1531,3 @@ class DummyOptionable1(Optionable):
     single_warning_dummy1 = assert_single_element(w)
     self.assertEqual(single_warning_dummy1.category, DeprecationWarning)
     self.assertEqual('vv', vals1.foo)
-
-
-# TODO: Figure out why this testing is necessary.
-class OptionsTestStringPayloads(OptionsTest):
-  """Runs the same tests as OptionsTest, but backed with `Config.loads` vs `Config.load`."""
-
-  def _create_config_from_strings(self, config):
-    with io.BytesIO(b'') as fp:
-      self._write_config_to_file(fp, config)
-      fp.seek(0)
-      payload = fp.read()
-      return Config.load_file_contents(
-        config_payloads=[FileContent(path='blah', content=payload)],
-      )
```

Comment:
This wasn't doing anything differently than the normal `OptionsTest`. It had two issues:

1) this function `_create_config_from_strings` was never called by any tests
2) `config_payloads` on line 1545 is an invalid kwarg

I removed this because it does not seem necessary and it's unclear why Kris added this in 2018.